src/services/color/color.service.ts

```diff
@@ -202,8 +202,124 @@ export class ThemeColor {
 }
 
 export const colorSets = {
-    keppel: require('../../data/keppel-colors.json'),
-    microFocus: require('../../data/micro-focus-colors.json')
+    keppel: {
```

Comment:
Note - had to change this as we can no longer use requires in our components as this is not supported by angular compiler. Requires will still work elsewhere in our docs as they are built by webpack.